extensions/jaeger/runtime/pom.xml

```diff
@@ -33,6 +33,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <!-- transitive dependency from jaeger-thrift, consider removing once we move to jaeger 1.x -->
+        <dependency>
```

Comment:
I am missing smth here does Jaeger 0.34.x uses this?

Reply:
The javax version is part of the dependencies and has been excluded in the first PR. It's just bringing it back the Jakarta equivalent.